dev-python/terminaltables/terminaltables-3.1.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python{3_6,3_7} )
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
I can't add this, because the package fails tests for me. There's a bug open, but there's no way to indicate whether it works or not.
https://bugs.gentoo.org/723590

Reply:
I use it from my local overlay and didn't step into this, strange. That said `pgcli` does not seem to set the terminal title.

Reply:
I have the same failure when I run `terminaltables` tests, but upstream looks quite dead (last commit was more than three years ago).